src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -174,57 +175,69 @@ def create_repos_and_code_mappings(
 
     organization_id = organization_integration.organization_id
     for code_mapping in code_mappings:
-        repository = (
```

Comment:
Moving all of the logic from this loop to `process_code_mapping`. It will make `create_configurations` shorter and easier to read.